tests/test_utils.py

```diff
@@ -72,6 +72,16 @@ def test_get_config_no_distutils(tmpdir):
             "username": "testuser",
             "password": "testpassword",
         },
+        "testpypi": {
```

Comment:
We should consolidate on `testpypi`. If our docs are using pypitest, we should consolidate

Reply:
Documentation is saying `testpypi` but the existing code in `utils.py` was using `pypitest`. I didn't want to break backwards compatibility, but would be happy to remove all references to `pypitest` if you want.

Reply:
I'd say let's take this opportunity to consolidate on `testpypi`.

Reply:
:+1: